Add feast integration demo

FelipeAdachi opened the pull request:
## Description

Add feast-whylogs integration demo in `examples`:
- Jupyter notebook
- Sample parquet data files
- Images
- Feast infra files

### General Checklist

* [ ] Tests added for this feature/bug
      if it was a bug, test must cover it.
* [ ] Conform by the style guides, by using formatter
* [ ] Documentation updated
* [ ] (optional) Please add a label to your PR

FelipeAdachi committed:
Add feast integration demo

FelipeAdachi committed:
isort

jamie256 reviewed (changes requested):
Can we move the data (parquet files) out of the whylogs repo? I know its relatively small but it would be better to separate these.

FelipeAdachi committed:
- Removed parquet files from the repo and moved to s3
- Changes in code to make notebook pass test
- Changed manner of materializing, to make the notebook run faster
- Added descriptions

FelipeAdachi committed:
debug notebook test

FelipeAdachi committed:
add .gitkeep to keep data folder

jamie256 reviewed (approved):
LGTM!
